samples/bluetooth/peripheral_nfc_pairing/sample.yaml

```diff
@@ -5,5 +5,6 @@ tests:
   samples.bluetooth.peripheral_nfc_pairing:
     build_only: true
     build_on_all: true
-    platform_allow: nrf52840_pca10056 nrf52_pca10040 nrf5340pdk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp
+    platform_allow: nrf52840_pca10056 nrf52_pca10040 nrf5340pdk_nrf5340_cpuapp nrf5340pdk_nrf5340_cpuappns
```

Comment:
nrf52840_pca10056 nrf52_pca10040 -> nrf52840dk_nrf52840 nrf52dk_nrf52832

Reply:
Thanks for noticing. Done